src/bin/stat.rs

```diff
@@ -0,0 +1,54 @@
+#![deny(warnings)]
+
+extern crate coreutils;
+extern crate extra;
+extern crate syscall;
+
+use std::env;
+use std::io::{stdout, stderr, Write};
+use coreutils::ArgParser;
+use extra::option::OptionalExt;
+
+const MAN_PAGE: &'static str = /* @MANSTART{chmod} */ r#"
```

Comment:
I'm just passing through, so I have no real idea what I'm looking at, but should this be `@MANSTART{stat}` instead of `@MANSTART{chmod}`?